### ISSUE TITLE
data : Server Side loading

### ISSUE THREAD

**Issue by miamilabs** [pavankataria/SwiftDataTables#62]
Hi Community,

first of all, i come from different world (Java Coding...) but i am also playing around with swift and trying to get into this. So i am not expert at all here. Excuse me if my question is wrong here.

Question : 

Does this Project support Server Side loading of data?

It would be great to have also pagination (https://github.com/pavankataria/SwiftDataTables/issues/57) as there are some data which have 3000+ rows with a lot of information.

Thank you

**Comment by pavankataria** [comment]
Yes it does though not as smoothly as I'd like it to. You have to refresh the entire table in order to support dynamic datasources.
This is an area that can be improved but because there are other priorities for this library but the process of making this seamless is on the backlog.

**Comment by tranquangdat1397** [comment]
Has this feature been implemented yet?

**Comment by pavankataria** [comment]
This is now available after having implemented a diffing algorithm allowing you to append to your dataset using the new setData API. 
This will be available in an upcoming release!

**Comment by pavankataria** [comment]
Hey, this has been fixed in v0.9.0. Use `setData(_:animatingDifferences:)` to load data asynchronously after table creation.

```swift
let table = SwiftDataTable(columns: columns)
// Later, after async fetch:
table.setData(fetchedData, animatingDifferences: true)
```
You can go to [Working with Data](https://pavankataria.github.io/SwiftDataTables/documentation/swiftdatatables/workingwithdata) for examples.
